Spec/RestClient.swift

```diff
@@ -173,8 +173,20 @@ class RestClient: QuickSpec {
                     
                     expect(testHTTPExecutor.requests.first?.URL?.host).toEventually(equal("fake.ably.io"), timeout: testTimeout)
                 }
+
+                it("should ignore an environment when host is not in production") {
```

Comment:
@tcard perhaps you should just take the PR and reword as you see fit as @ricardopereira is away on holiday for a little while still